openpype/pipeline/workfile/path_resolving.py

```diff
@@ -408,6 +408,9 @@ def get_custom_workfile_template(
     # add root dict
     anatomy_context_data["root"] = anatomy.roots
 
+    # extend anatomy context with os.environ
+    anatomy_context_data.update(os.environ)
```

Comment:
At the very least I'd reverse the logic so that if any anatomy context data would never be overwritten if one were to match a key in `os.environ` by accident. Just to foolproof it?

Reply:
done with that, i just got os.environ and updated it with anatomy afterwards. check it out!